### PR TITLE
add dockerfile for building image via PRs

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton you want to hook CI up to this repo and test this PR as well?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM openshift/origin-release:golang-1.10
+COPY . /go/src/github.com/openshift/cluster-samples-operator
+RUN cd /go/src/github.com/openshift/cluster-samples-operator && \
+    go build ./cmd/cluster-samples-operator
+
+FROM centos:7
+RUN useradd cluster-samples-operator
+USER cluster-samples-operator
+COPY --from=0 /go/src/github.com/openshift/cluster-samples-operator /usr/bin
```
Comment (gabemontero) [inline Dockerfile]: we need to COPY the content from https://github.com/openshift/cluster-samples-operator/tree/master/tmp/build/assets/arch as well

Comment (bparees) [reply Dockerfile]: thanks, fixed

**Comment by smarterclayton** [comment]
Yes

**Comment by gabemontero** [comment]
looks like the prow stuff held up ... shall we LGTM it @bparees ?

**Comment by bparees** [comment]
guess so.

**Pull request merged**